src/main/java/org/apache/maven/reporting/exec/DefaultMavenPluginManagerHelper.java

```diff
@@ -34,6 +32,10 @@
 import org.codehaus.plexus.component.annotations.Component;
 import org.codehaus.plexus.component.annotations.Requirement;
 import org.codehaus.plexus.logging.Logger;
+import org.eclipse.aether.RepositorySystemSession;
+import org.eclipse.aether.graph.DependencyFilter;
+import org.eclipse.aether.repository.RemoteRepository;
+import org.eclipse.aether.util.filter.ExclusionsDependencyFilter;
```

Comment:
Can we list on a wiki page the list of missing APIs ? If the goal is to hide maven-resolver in a clean maven-core provided api, we need to list the bits that are currently missing.  The fact that the `MavenPluginManager` is depending on the aether api is wrong.

Reply:
Yes, let's collect "missing APIs", agreed.

OTOH, this PR is to "up component maven dep to maven 3.2.x", that does not provide those. 

IMHO, we could start collecting and providing them in Maven 4.x line (or even maybe Maven 3.9.x line), and when we start doing plugins 4.x line (like we did with plugins 3.x that were aligned with Maven 3.x), then we could move all plugins to newly published APIs... 

Created page https://cwiki.apache.org/confluence/display/MAVEN/Maven+Missing+APIs